CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ https://azure.microsoft.com/updates/azure-basic-load-balancer-will-be-retired-on
 
 * Features:
   * AKS patch versions 1.28.14, 1.29.9, 1.30.5 are now available. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information.
-  * AKS version [`1.31`](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/) is now generally available. Please check the release tracker for when your region will receive the GA update. Some regions may not receive this update until later in November.
+  * AKS version [`1.31`](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/) will be generally available by the end of November. Version availability for each region can be determined by following the steps [here](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions).
```

Comment:
Think for checking version availability they would have to rely on Az aks get-versions so its better to cut to the specific link within this doc - https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#azure-portal-and-cli-versions